docs/Makefile

```
@@ -1,31 +1,74 @@
+	@if [ -d site-base ]; then \
+		echo "site-base exists, skipping (rm -rf site-base to rebuild)"; \
+	else \
+		echo "Building base site from $(BASE)..."; \
+		WORKTREE=$$(mktemp -d); \
+		trap 'git -C .. worktree remove --force "$$WORKTREE" 2>/dev/null' EXIT; \
+		git -C .. worktree add --detach "$$WORKTREE" "$(BASE)"; \
+		cd "$$WORKTREE/docs" && uv run --with mkdocs-material --with mkdocs-mermaid2-plugin --with mkdocs-glightbox mkdocs build -d "$(CURDIR)/site-base"; \
+	fi
+
+.PHONY: compare-build-head
+compare-build-head:
```

checkmake (0.3.2)

[warning] Target body for "compare-build-head" exceeds allowed length of 5 lines (7).

(maxbodylength)